algebra-core/Cargo.toml

```diff
@@ -42,3 +43,4 @@ std = []
 parallel = [ "std", "rayon" ]
 derive = [ "algebra-core-derive" ]
 asm = []
+stable = []
```

Comment:
A better approach is to use something like this: https://crates.io/crates/select-rustc, and gate it so that the attribute is true only on `nightly`, so maybe something like this: `#[cfg_attr(feature = "asm", rustversion::nightly)]`, or perhaps this: `#[cfg_attr(feature = "asm", rustversion::attr(nightly, feature(asm)))]`

This is because features are supposed to be additive, i.e. features are supposed to form a poset: https://github.com/rust-lang/cargo/issues/4328

Reply:
Unfortunately, it appears non built-in inner attributes are unstable. Select-rust is only for outer attributes.